public/app/features/explore/TraceView/components/TraceTimelineViewer/SpanLinks.tsx

```diff
@@ -100,6 +100,33 @@ const renderMenuItems = (
           ))}
         </MenuGroup>
       ) : null}
+      {!!links.otherLinks?.length ? (
```

Comment:
Considering these sections are basically the same with just some string changed, we could probably deduplicate it a bit in some separate PR @joey-grafana

Reply:
I plan on putting more time into this PR to add tests to createLinkSpan, I can deduplicate it in this PR once the traces squad is feeling okay with the UX changes I've made to links.

Reply:
sorry didn't mean this needed to be done in this PR, honestly by you :). It was mainly just a note for us but obviously would not mind if you want to do it here :)

Reply:
I know I dont need to but I will :D